hebiros_advanced_examples/package.xml

```diff
@@ -12,7 +12,7 @@
   <url type="repository">https://github.com/HebiRobotics/HEBI-ROS.git</url>
   <url type="bugtracker">https://github.com/HebiRobotics/HEBI-ROS/issues</url>
 
-  <author email="x@hebirobotics.com">Xavier Artache</author>
+  <author email="sami@hebirobotics.com">Sami Mian</author>
```

Comment:
Typically (I think), you'd just add an author, but update the maintainer.  That way, you give credit for the past work on the project.